backend/src/main/java/com/huellapositiva/infrastructure/orm/entities/JpaCredential.java

```diff
@@ -46,4 +48,12 @@
 
     @Column(name = "surname")
     private String surname;
+
+    @Column(name = "hash_recovery_password", unique = true)
+    private String hashRecoveryPassword;
+
+    //@CreationTimestamp
```

Comment:
Why not delegate the creation to the DB engine?

Reply:
Fixed it.